app/helpers/alchemy/admin/essences_helper.rb

```diff
@@ -9,6 +9,17 @@ module EssencesHelper
       # Renders the Content editor partial from the given Content.
       # For options see -> render_essence
       def render_essence_editor(content, options = {}, html_options = {})
+        if !options.empty?
+          Alchemy::Deprecation.warn <<~WARN
+            Passing options to `render_essence_editor` has been removed.
+            Add your static settings to the #{content.name} content definition in `config/alchemy/elements.yml`.
+          WARN
+        end
+        if !html_options.empty?
+          Alchemy::Deprecation.warn <<~WARN
```

Comment:
I think this should raise an error so people actually fix their stuff before going live with a new site.